src/resources/lang/ru/permissions.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+return [
+    // Character permissions
+    'character_paps_label' => 'Paps игрока',
+    'character_paps_description' => '',
+
+    // Corporation permissions
+    'corporation_paps_label' => 'Paps корпорации',
+    'corporation_paps_description' => '',
```

Comment:
Please remove untranslated captions - provided a blank caption will render it as it - meaning, the caption will not be displayed. If the caption is not translated, we will search for the default one - in english.

Reply:
Its work for me. We use "Paps" in Russian like in English.